auxiliary_modules/storage_access/yaml_api_test.py

```diff
@@ -0,0 +1,19 @@
+from .yaml_api import parse_yaml
+
+YAML_CONTENT = """
+dirs:
+  raw_data: raw_data
+  processed_data: processed_data
+  schemas: schemas
+    """
+
+
+def test_yaml_api(tmp_path):
```

Comment:
`tmp_path` is a [fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html) provided by pytest. A very useful one I must admit. It helped me to avoid creating new directories manually.

Reply:
👍